### ISSUE TITLE
Symbol action may convert failed in saga

### ISSUE THREAD

**Issue by purepennons** [zalmoxisus/redux-devtools-extension#542]
I modified the [saga-counter](https://github.com/purepennons/redux-devtools-extension/tree/master/examples/saga-counter) example to use Symbol action instead.

And when I triggered multi times (maybe fast), the devtool caught  all the actions, but some of them were converted failed.

```js
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ serialize: true }) || compose;
```

```js
export const INCREMENT = Symbol('INCREMENT')
export const INCREMENT_IF_ODD = Symbol('INCREMENT_IF_ODD')
export const INCREMENT_ASYNC = Symbol('INCREMENT_ASYNC')
export const DECREMENT = Symbol('DECREMENT')
```

![screen shot 2018-07-31 at 1 22 13 pm](https://user-images.githubusercontent.com/4588113/43439383-905ae3c4-94c5-11e8-974a-5b16aec32bd9.png)

**Comment by zalmoxisus** [comment]
Related to https://github.com/kolodny/jsan/issues/24

**Comment by zalmoxisus** [comment]
It was downgraded and fixed in `2.16`, which will land in a day on Chrome Store.
